tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1938,7 +1938,11 @@ def assignDerivedParameters(state):
     computeBytes = state["ProblemType"]["ComputeDataType"].numBytes()
     state["_GlobalAccumulation"] = None
     computeName  = state["ProblemType"]["ComputeDataType"].toName()
-    if state["GlobalSplitUAlgorithm"] == 'SingleBuffer':
+    if state["StreamK"] > 0 and state["StreamKAtomic"] == 0:
+      # StreamK Workspace size
+      computeBytes = state["ProblemType"]["ComputeDataType"].numBytes()
```

Comment:
This line is duplicated (same as line 1938)

Reply:
Fixed